Sources/llbuild2fx/SpawnProcess.swift

```diff
@@ -365,3 +365,10 @@ public struct SpawnProcessResult: FXValue, FXTreeID {
         exitCode = codableValue
     }
 }
+
+extension SpawnProcessResult: Codable {
+    private enum CodingKeys: String, CodingKey {
+        case treeID
```

Comment:
I noticed in `SpawnProcessResult` we have
```
    public var codableValue: Int32 {
        exitCode
    }
```
but the `Codable` conformance that I added in this PR also includes `treeID`.. is this desirable?

Reply:
Definitely not.   The treeID should not be ending up in the codableValue.   It must be stored in the refs.